references/sdk_schemas.md

```diff
@@ -39,6 +39,8 @@ the 'paths' field.
 
 ## DatasetSpec
 Specification for creating a dataset
+
+note : metadata field does not accept strings containing '__' as dict key
```

Comment:
I've added some insight here so that this specification is in the user doc, but note sure this is the right way/place. I feel this is a bit redundant for each asset. If someone has a better idea of where to mention this new constraint in the doc, that would be nice.

Reply:
did you re-generate this file with the`make doc` command? It is created automatically from the client functions docstring
unfortunately I don't see how to not repeat it for every asset

Reply:
Yes I only edited the docstring in schemas.py, this was generated by `make doc`.